src/Ui/RestBase.php

```diff
@@ -135,6 +141,6 @@ protected function buildResponseStream(ServerRequestInterface $request): ?Stream
 
     protected function handleNativeException(CoronadoException $e, $request): ?StreamInterface
     {
-        return $this->streamFactory->createStream('An error occured: ');
```

Comment:
This actually breaks the base class as it returns neither null nor StreamInterface. Should declare it as abstract method in this case